src/lock.stanza.disabled

```diff
@@ -5,12 +5,13 @@ defpackage slm/lock:
   import semver
   import toml
 
-  import slm/logging
-  import slm/utils
-  import slm/file-utils
   import slm/dep-types
-  import slm/task
+  import slm/errors
+  import slm/file-utils
   import slm/flags
+  import slm/logging
+  import slm/task
+  import slm/utils
```

Comment:
Maybe we just remove this file completely ?

Reply:
I was hoping to leave it in there if anyone wanted to replace the functionality, but it will still remain in the git history so I'm fine with removing it.